src/regexp.c

```diff
@@ -617,11 +619,14 @@ static Renode *parserep(Restate *g) {
 
         atom = parseatom(g);
         if (g->lookahead == L_COUNT) {
-                int min = g->yymin, max = g->yymax;
+                unsigned char min = g->yymin, max = g->yymax;
                 next(g);
+                if ((g->yymin & ~(0xff)) || (g->yymax & ~(0xff)))
+                        die(g,
+                            "invalid quantifier: min & max should be in "
+                            "[0-255]");
                 if (max < min)
                         die(g, "invalid quantifier");
-                return newrep(g, atom, re_accept(g, '?'), min, max);
```

Comment:
Should this line really be removed?

Reply:
Apologies, it's been a while, and I've lose context. I'll close this PR.